pkgs/development/web/esbuild/default.nix

```diff
@@ -0,0 +1,23 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "esbuild";
+  version = "0.11.12";
+
+  src = fetchFromGitHub {
+    owner = "evanw";
+    repo = "${pname}";
+    rev = "v${version}";
+    sha256 = "1mxj4mrq1zbvv25alnc3s36bhnnhghivgwp45a7m3cp1389ffcd1";
+  };
+
+  vendorSha256 = "1n5538yik72x94vzfq31qaqrkpxds5xys1wlibw2gn2am0z5c06q";
+
+  doCheck = false;
```

Comment:
Why did you disable tests?

Reply:
This is just a leftover because I based this on [minify](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/web/minify/default.nix), tests work fine.